static/src/stylesheets/module/content/_interactive.scss

```diff
@@ -163,4 +128,17 @@
     [data-alt="Live stream"] {
         @extend %u-responsive-ratio;
     }
+
+    .content__meta-footer {
+        .meta__email,
+        .meta__twitter {
+            display: inline !important;
+        }
```

Comment:
Same again, not vital but above in the file you've left an empty line between selectors, whereas here their isn't. eww